discord/api/websocket.py

```diff
@@ -41,6 +46,11 @@ def __init__(self, client, token: str) -> None:
         self.heartbeat_acked = True
         self.closed: bool = False
 
+        self.guild_cache = LFUCache[Snowflake, Guild](1000)
+        self.channel_cache = LFUCache[Snowflake, BaseChannel](5000)
+        self.member_cache = LFUCache[Snowflake, dict](5000)
+        self.user_cache = LFUCache[Snowflake, dict](5000)
```

Comment:
There should also be a message cache with the size of probably 2k but we'll decide

Reply:
I don't think using the LFU strategy for caching messages is a good idea. Maybe `collections.deque` instead?

Reply:
We will make our own FIFO cache to store the messages, it will subclass `collections.OrderedDict`

Reply:
Added a message cache with the capacity of 1000 messages